packages/vite/src/node/server/environment.ts

```diff
@@ -117,12 +122,21 @@ export class DevEnvironment extends BaseEnvironment {
       this.pluginContainer!.resolveId(url, undefined),
     )
 
-    this.hot = context.hot || createNoopHotChannel()
```

Comment:
Is `context.hot: boolean` used somewhere?

Reply:
It wasn't used anymore, removed 👍